app/models/project_user_resource.rb

```diff
@@ -18,13 +21,13 @@ class ProjectUserResource < ApplicationRecord
                        .sum(:man_month)
                        .round(Settings.digits.length_2)
   }
-
   scope :total_man_month_projects, lambda {|project_ids, month, year|
     project_user_ids = ProjectUser.where(project_id: project_ids).pluck(:id)
     where(project_user_id: project_user_ids)
       .total_man_month_year(month, year)
       .round(Settings.digits.length_2)
   }
+  scope :sorted_by_month_and_year, ->{order(year: :asc, month: :asc)}
```

Comment:
```suggestion
  scope :sorted_by_month_and_year, ->{order :year, :month}
```
asc là mặc định nên xóa dc